airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -19,6 +19,20 @@
 Changelog
 ---------
 
+2.0.2
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix using XCom with &#39;&#39;KubernetesPodOperator&#39;&#39; (#17760)``
```

Comment:
I guess the log is auto generated by script? It seems that the the code quotation char isn't parsed correctly

Reply:
Yep. I will fix it (and we are likely going a bit different direction anyway - trying the towncrier soon).